tests/build/build.go

```diff
@@ -412,25 +415,46 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 					return nil
 				}, timeout, constants.PipelineRunPollingInterval).Should(Succeed(), fmt.Sprintf("timed out when waiting for the PipelineRun to start for the component %s/%s", testNamespace, componentName))
 			})
+			It("should lead to a PaC init PR creation", func() {
```

Comment:
When we update the PaC init branch then why do we checking init PR creation again? Is it needed here?

Reply:
Updated the description.

Reply:
I didn't mean the description.. do we need to check again the PR creation when we are updating the branch.. Isn't it redundant step?

Reply:
I don't think so because it needs to get the latest commit SHA after updating the branch

Reply:
we are not doing any extra validation in this step apart from reading the prHeadSha .. which is used in later step.. do we need a verification in this step like `prHeadSha` (after update) != `prHeadSha` (at pr creation) ?

Reply:
thank you for the suggestion. fixed.